ext/travisci/install_puppetserver.sh

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env bash
+
+lein=lein2
+
+# Update a single dependency in a leiningen project.clj file.
+update_dependency_var() {
+    local file="$1"
+    local varname="$2"
+    local new_version="$3"
+
+    SED_ADDRESS="(def $varname"
+    SED_REGEX="\".*\""
+    SED_REPLACEMENT="\"$new_version\""
+    SED_COMMAND="s|$SED_REGEX|$SED_REPLACEMENT|"
+
+    set -e
+    set -x
+
+    sed -i -e "/$SED_ADDRESS/ $SED_COMMAND" $file
+}
+
+lein-pprint() {
+    export PUPPETSERVER_HEAP_SIZE=1G
```

Comment:
It didn't work. And yes we need it for pprint, since that loads the puppetserver jar, which will complain straight to stdout if this isn't set, making the version number non-parsable.